StringReader.py

```diff
@@ -0,0 +1,6 @@
+#This will read a string in and spit it right back out.
+
+print("Enter a string.")
```

Comment:
If you add a comma, I think it makes it so it leaves it on the same line. Want to try it?

Reply:
Sure.  Give me a moment.

Reply:
I just realized I was giving you bad advice. The correct way to avoid a newline is to specify `end = ''` as the second parameter to `print`.

Incidentally, when talking about code, it is conventional to surround it with backticks, which is how I got it to look nice in the previous paragraph.